fulp_modules/_defines/species_defines.dm

```diff
@@ -34,7 +34,7 @@
 /**
  * Beefman trauma pref defines
  */
-#define BEEF_TRAUMA_STRANGERS "Strangers"
-#define BEEF_TRAUMA_HALLUCINATIONS "Hallucinations"
-#define BEEF_TRAUMA_OCKY_ICKY "Ocky Icky"
-#define BEEF_TRAUMA_HYPNOTIC_STUPOR "Hypnotic Stupor"
+#define BEEF_TRAUMA_STRANGERS "Phobia of Strangers"
+#define BEEF_TRAUMA_HALLUCINATIONS "Severe Hallucinations"
+#define BEEF_TRAUMA_OCKY_ICKY "Phobia of Conspiracies"
+#define BEEF_TRAUMA_PACIFISM "Pacifism"
```

Comment:
1. Rename the define
2. Do we really want pacifism as a trauma? Surely there's some other trauma that can serve as a replacement that doesn't make a quirk useless.

Reply:
Changed it to monophobia and renamed the defines